scilpy/io/streamlines.py

```diff
@@ -23,3 +28,63 @@ def check_tracts_same_format(parser, tractogram_1, tractogram_2):
     if not ext_1 == ext_2:
         parser.error(
             'Input and output tractogram files must use the same format.')
+
+
+def load_trk_in_voxel_space(trk_file, anat=None, grid_res=None,
+                            raise_on_empty=True):
+    """
+    Load streamlines in voxel space, corner aligned
+
+    :param trk_file: path or nibabel object
+    :param anat: path or nibabel image (optional)
+    :param grid_res: specify the grid resolution (3,) (optional)
+    :param raise_on_empty: specify if an error should be raised when
+           the file is empty
+    :return: NiBabel tractogram with streamlines loaded in voxel space
+    """
+    if isinstance(trk_file, six.string_types):
+        trk_file = nib.streamlines.load(trk_file)
+
+    if trk_file.header[Field.NB_STREAMLINES] == 0 and raise_on_empty:
+        raise Exception("The file contains no streamline")
+
+    if anat and grid_res:
+        raise Exception("Parameters anat and grid_res cannot be used together")
+
+    if anat:
+        if isinstance(anat, six.string_types):
+            anat = nib.load(anat)
+        spacing = anat.header['pixdim'][1:4]
+    elif grid_res:
+        spacing = grid_res
+    else:
+        spacing = trk_file.header[Field.VOXEL_SIZES]
+
+    affine_to_voxmm = get_affine_rasmm_to_trackvis(trk_file.header)
+    tracto = trk_file.tractogram
+    tracto.apply_affine(affine_to_voxmm)
+    streamlines = tracto.streamlines
+
+    if trk_file.header[Field.NB_STREAMLINES] > 0:
+        streamlines._data /= spacing
+    return streamlines
+
+
+def save_from_voxel_space(streamlines, anat, ref_tracts, out_name):
```

Comment:
to remove, use dipy.io.streamlines - load_tractogram and save_tractogram instead